### ISSUE TITLE
didnt even work now?

### ISSUE THREAD

**Issue by mazmamad** [ivanstepanovftw/dota-camera-changer#3]
i try it in windows and nothing happend :(

**Comment by ivanstepanovftw** [comment]
Yeah, its not working right now. I have played updated Dota 2 recently, and its camera already fine for me

**Comment by ivanstepanovftw** [comment]
There are some suggestions from issue #2 , you can try it
